### ISSUE TITLE
Allow users to magnify beyond 200% and maintain function of the app

### ISSUE THREAD

**Issue by ajpeddakotla** [hypothesis/product-backlog#788]
This issue is part of the larger accessibility epic written up here #753

1.4.4: Resize text
- [ ] Allow users to magnify beyond 200% and maintain function of the app (A)

Score given: 6

Problem:
Users should be able to increase the size of text to 200% without loss of content or
functionality and without requiring horizontal scrolling. Currently the content resizes well, however when increased to 200%, it obscures the article content as well as the show/hide toggle. This makes it difficult to interact with for users who require magnification. See W3C guidelines for this here: https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-scale.html

<img width="437" alt="screen shot 2018-08-31 at 2 24 25 pm" src="https://user-images.githubusercontent.com/7463523/44932086-97fced80-ad29-11e8-84f5-9d9ce83e8e34.png">

Acceptance Criteria:
- [ ] User should be able to magnify and still maintain functionality of the app

**Comment by ajpeddakotla** [comment]
| User Engagement | Building Community | Level of Effort |
|---|---|---|
| 3 | 1 | 1 |

**Comment by robertknight** [comment]
This issue may need to be resolved, at least initially, with documentation rather than a product change. The issue is that in the context of a PDF, there are two kinds of zoom possible:

1. Zooming the _PDF contents_
2. Zooming the _user interface_, which includes all the PDF viewer controls (eg. the toolbar at the top) and Hypothesis

If you press <kbd>Cmd</kbd> + <kbd>+/-</kbd> on a normal web page you will end up zooming the whole page. In a PDF however, the PDF viewer intercepts this to zoom just the document contents. To zoom the user interface you need to use the browser's menus:

<img width="1336" alt="Screenshot 2019-12-18 12 01 49" src="https://user-images.githubusercontent.com/2458/71084904-323aac00-218e-11ea-9179-8e2c447d77ce.png">

The Hypothesis UI works when the UI zoom is set to 200%, so it is possible we can write some documentation and then close the issue.

Being able to zoom the content of a PDF is IMO critical functionality so I don't think we should change that. We could potentially show a hint to the user when they use the zoom keys. I think I would like to see some evidence that this really is a confusing problem for affected users before. doing that though.

**Comment by ajpeddakotla** [comment]
@klemay what do you think the next steps here should be? This is one of the issues we need to address to meet level II standards. I do agree with @robertknight's assessment above, that we could resolve this via some more documentation, but I'm not sure if that meets WCAG standards.

**Comment by klemay** [comment]
@ajpeddakotla Rob's assessment is correct. Rather than closing this issue, I'm going to move this to the support repo so we know to create the KB article.
